src/main/resources/templates/start.html

```diff
@@ -2,29 +2,40 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=divice-width, initial-scale=1, shrink-to-fit=no">
     <title>BulkBuy</title>
 
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
     <script type="text/javascript">
         document.getElementById("SearchOrder").onclick = function () {
             location.href = "orderLookup.html";
         };
     </script>
 
 </head>
-<body>
-BulkBuy Dashboard
+<body class="bg-secondary">
+<div class="d-flex justify-content-center">
+    <div class="header__dashboard" style="display: grid">
+        <h1 class="display-1" style="color: white">Welcome To BulkBuy</h1>
+        <br style="height: 10px">
+        <br style="height: 10px">
```

Comment:
- Prefer CSS margin and padding over multiple br tags.
- Avoid inline CSS styles at all cost.  Use linked styles instead, for a consistent theme and less bloat.